CSharp/README.md

```diff
@@ -23,6 +24,115 @@ var httpClient = new HttpClient();
 IUserService userService = new UserService();
 ```
 
+### Don't use explicit *this* reference
+
+It is redundant and does not add for readability. For more information read [this SO post](http://stackoverflow.com/questions/5885249/to-use-or-not-to-use-the-this-qualifier-in-c-sharp).
+
+```csharp
+// Bad
+this.ValidateParameters();
+
+// Good
+ValidateParameters();
+```
+
+### Use *string* instead of *String* keyword

```

Comment:
> but not sure about doing the .editorconfig, .vssetting and .ruleset thing now

Sure, I said

> The end result of **this guide**

Not this PR. We already have those files defined on BetterCare, so it will be a matter of grabbing them and add a few comments.